apps/openmw/mwworld/worldimp.hpp

```diff
@@ -103,25 +103,39 @@ namespace MWWorld
             std::unique_ptr<MWWorld::WeatherManager> mWeatherManager;
             std::shared_ptr<ProjectileManager> mProjectileManager;
 
-            bool mGodMode;
-            bool mScriptsEnabled;
+            bool mSky = true;
+            bool mGodMode = false;
+            bool mScriptsEnabled = true;
             std::vector<std::string> mContentFiles;
 
             std::string mUserDataPath;
 
             osg::Vec3f mDefaultHalfExtents;
             bool mShouldUpdateNavigator = false;
 
-            // not implemented
-            World (const World&);
-            World& operator= (const World&);
-
             int mActivationDistanceOverride;
 
+            std::string mStartCell;
+
+            float mSwimHeightScale;
+
+            float mDistanceToFacedObject = -1;
```

Comment:
I think this syntax:
```cpp
float mDistanceToFaceObject{-1.f};
```
is preferred in modern C++. List initialization is safer. In this case you do narrowing because "-1" is an integer value, copy initialization doesn't reject it, list initialization does. It's probably for the best to use list initialization for *any* type.

Reply:
FWIW, that's called uniform initialization, not list initialization. So called because it's a uniform syntax to initialize a POD type, array, struct, or call a constructor. I would definitely prefer it either way, though, since it protects against narrowing, and won't otherwise do implicit conversions, which can be a source of bugs.

Reply:
`=` is more obvious and readable, so I'm not sure this is a situation where the modern, safer way of doing things is beneficial.

Reply:
I suppose we need to define somewhere in contributing guidelines a way to init fields. So far OpenMW uses at least 4 different ways to init fields ("=", ":", "{}" and in constructor). As for uniform initialization, personally I do not like to use it for simple fields - it looks ugly.

Reply:
I rewrote this PR to use an old-style initialization for all WorldManager fields.